README.adoc

```diff
@@ -695,7 +698,7 @@ Inside most of our jobs, we used TypeScript written scripts to perform our actio
 
 We are using https://docs.sentry.io/[Sentry] to monitor production and review app's exceptions raised.
 
-A custom utility function, called `addRequestScopeToSentry`, is provided to format the data used for the report.
+Thanks to the https://docs.sentry.io/platforms/javascript/guides/nextjs/[Sentry Next.js SDK], the implementation is now way cleaner. We have added a wrapper around the `withSentry` middleware to be able to use is it with SSR.
```

Comment:
```suggestion
We're using https://docs.sentry.io/platforms/javascript/guides/nextjs/[Sentry Next.js SDK] and have added a wrapper around the `withSentry` middleware to be able to use is it with SSR.
```

Reading a README telling about an improvement is weird imho, since we don't read the previous versions.

Reply:
Make sense.